dipy/tracking/utils.py

```diff
@@ -568,6 +574,149 @@ def target(streamlines, target_mask, affine, include=True):
             yield sl
 
 
+def sl_near_roi(sl, x_roi_coords, tol, mode='any'):
+    """
+    Is a streamline near an ROI.
+
+    Implements the inner loops of the :func:`near_roi` function.
+
+    Parameters
+    ----------
+    sl : array, shape (N, 3)
+	    A single streamline
+    x_roi_coords : array, shape (M, 3)
```

Comment:
Please use the full name. transformed_roi or just say roi_coords or roi. Of course the roi has to be in the same space with the streamlines. x is misleading